src/findingCfg.js

```diff
@@ -0,0 +1,358 @@
+import CheckCircleOutlineOutlinedIcon from '@mui/icons-material/CheckCircleOutlineOutlined'
+import HelpOutlineOutlinedIcon from '@mui/icons-material/HelpOutlineOutlined'
+import ReportProblemIcon from '@mui/icons-material/ReportProblem'
+import ReportProblemOutlinedIcon from '@mui/icons-material/ReportProblemOutlined'
+
+import {
+  FILTER_SEMANTICS,
+  META_SPRINT_NAMES,
+} from './consts'
+import {
+  capitalise,
+  mostSpecificRescoring,
+  orderRescoringsBySpecificity,
+} from './util'
+import { artefactMetadataFilter } from './ocm/util'
+
+
+export const FINDING_TYPES = {
+  CODECHECKS_AGGREGATED: 'codechecks/aggregated',
+  DIKI: 'finding/diki',
+  LICENSE: 'finding/license',
+  MALWARE: 'finding/malware',
+  OS_IDS: 'os_ids',
+  SAST: 'finding/sast',
+  VULNERABILITY: 'finding/vulnerability',
+}
+Object.freeze(FINDING_TYPES)
+
+
+export const SUPPORTED_TYPES_FOR_RESCORINGS = [
+  FINDING_TYPES.LICENSE,
+  FINDING_TYPES.MALWARE,
+  FINDING_TYPES.VULNERABILITY,
```

Comment:
I think it is okay to keep hardcoded for now (same for findings types).
However, in future we should consider whether delivery-service could (and should) expose known finding-types and supported recoring types.